src/App.test.tsx

```diff
@@ -163,57 +175,57 @@ describe('App', () => {
     hasSession?: boolean
   }) {
     server.use(
-      rest.get('/internal/user', (_, res, ctx) => {
+      http.get('/internal/user', (info) => {
         if (hasSession) {
-          return res(ctx.status(200), ctx.json(internalUser))
+          return HttpResponse.json(internalUser, { status: 200 })
         } else {
-          return res(ctx.status(200), ctx.json({}))
+          return HttpResponse.json({}, { status: 200 })
         }
       }),
-      rest.get('/internal/users/current', (req, res, ctx) => {
-        return res(ctx.status(200), ctx.json({}))
+      http.get('/internal/users/current', (info) => {
+        return HttpResponse.json({}, { status: 200 })
       }),
-      graphql.query('DetailOwner', (_, res, ctx) =>
-        res(ctx.status(200), ctx.data({ owner: 'codecov' }))
+      graphql.query('DetailOwner', (info) =>
+        HttpResponse.json({ data: { owner: 'codecov' } }, { status: 200 })
       ),
-      graphql.query('CurrentUser', (req, res, ctx) => {
+      graphql.query('CurrentUser', (info) => {
         if (hasLoggedInUser) {
-          return res(ctx.status(200), ctx.data(user))
+          return HttpResponse.json({ data: user }, { status: 200 })
         }
-        return res(ctx.status(200), ctx.data({}))
+        HttpResponse.json({ data: {} }, { status: 200 })
       }),
-      graphql.query('GetPlanData', (req, res, ctx) => {
-        return res(ctx.status(200), ctx.data({}))
+      graphql.query('GetPlanData', (info) => {
+        return HttpResponse.json({ data: {} }, { status: 200 })
       }),
-      graphql.query('OwnerTier', (req, res, ctx) => {
-        return res(ctx.status(200), ctx.data({}))
+      graphql.query('OwnerTier', (info) => {
+        return HttpResponse.json({ data: {} }, { status: 200 })
       }),
-      graphql.query('Seats', (req, res, ctx) => {
-        return res(ctx.status(200), ctx.data({}))
+      graphql.query('Seats', (info) => {
+        return HttpResponse.json({ data: {} }, { status: 200 })
       }),
-      graphql.query('HasAdmins', (req, res, ctx) => {
-        return res(ctx.status(200), ctx.data({}))
+      graphql.query('HasAdmins', (info) => {
+        return HttpResponse.json({ data: {} }, { status: 200 })
       }),
-      graphql.query('owner', (req, res, ctx) => {
-        return res(
-          ctx.status(200),
-          ctx.data({ me: { owner: { isAdmin: true } } })
+      graphql.query('owner', (info) => {
+        return HttpResponse.json(
```

Comment:
Codecov is showing me line coverage for this file, which is unexpected.

Reply:
Yeaaa the test files were being included in the coverage reporting, this should be fixed. It was also causing downstream issues